.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: rust
 sudo: false
 cache: cargo
-rust: nightly
-os:
-- linux
-- osx
+rust:
+- stable
+- beta
```

Comment:
Up to you, but I suspect that we'd catch it on nightly if something was going to break?

Reply:
I _think_ it's helpful, mainly in case a breaking change makes it into beta but is fixed in nightly? Like if we always succeed betas, we will always succeed on the next stable release - and that guarantee isn't necessarily there for nightly rust.

Not super needed, but it could be nice to have.